client-v2/src/main/java/com/clickhouse/client/api/internal/HttpAPIClientHelper.java

```diff
@@ -601,6 +601,15 @@ public static Map<String, String> parseUrlParameters(URL url) {
         Map<String, String> params = new HashMap<>();
 
         try {
+            String path = url.getPath();

```

Comment:
just a note - this method should be outside of this class because I want to extract most common things.